### DISCUSSION TITLE
Configuration.proxyEnabled is actual?

### DISCUSSION THREAD

**Discussion by jazzboy65** [selenide/selenide#2289] [Q&A]
@asolntsev сan you please tell me how to download a file without a link and the href attribute, the only solution is to download files using the proxy server built into Selenide, or is there something else new?

**Comment by asolntsev** [comment]
There is 3 methods to download a file in Selenide: `HTTPGET` (requires `href` attribute), `PROXY` and `FOLDER` (the newest one).
All 3 work well. 

See
1. https://selenide.org/2019/12/10/advent-calendar-download-files/
2. https://selenide.org/2020/07/08/selenide-5.13.0/#new-file-download-mode-folder
3. https://youtu.be/GwHG550moGc
